components/AirdropDashboard.vue

```diff
@@ -0,0 +1,197 @@
+      <IconDiverMini class="text-airdrop-gold" />
+      <div
+        :class="[
+          'flex',
+          'my-[12px]',
+          'items-end',
+        ]"
+      >
+        <Label
+          :class="[
+            'font-bold',
+            'text-black',
+          ]"
+          :text="`${decay.factor}`"
+          preset="h1"
+        />
+        <Label
+          :class="[
+            'font-bold',
+            'text-black',
+            'ml-[8px]',
+            'mb-[4px]',
+          ]"
+          text="%"
+          preset="h2"
+        />
+      </div>
+      <Label
+        :class="[
+          'font-bold',
+          'text-medium-gray',
+        ]"
+        :text="$t('AirDrop.label.decay')"
+        preset="p6"
+      />
+    </div>
+    <div
+      :class="[
+        'flex',
+        'flex-col',
+        'items-start',
+        'justify-between',
+        'flex-grow',
+      ]"
+    >
+      <IconDiverMini class="text-airdrop-gold" />
+      <div
+        :class="[
+          'flex',
+          'my-[12px]',
+          'items-end',
+        ]"
+      >
+        <Label
+          :class="[
+            'font-bold',
+            'text-black',
+          ]"
+          :text="decay.days"
+          preset="h1"
+        />
+        <Label
+          :class="[
+            'font-bold',
+            'text-black',
+            'mr-[8px]',
+            'mb-[4px]',
+          ]"
+          :text="$t('AirDrop.mission.decay.days')"
+          preset="h2"
+        />
+        <Label
+          :class="[
+            'font-bold',
+            'text-black',
+          ]"
+          :text="decay.hours"
+          preset="h1"
+        />
+        <Label
+          :class="[
+            'font-bold',
+            'text-black',
+            'mr-[8px]',
+            'mb-[4px]',
+          ]"
+          :text="$t('AirDrop.mission.decay.hours')"
+          preset="h2"
+        />
+        <Label
+          :class="[
+            'font-bold',
+            'text-black',
+          ]"
+          :text="`${decay.minutes}`"
+          preset="h1"
+        />
+        <Label
+          :class="[
+            'font-bold',
+            'text-black',
+            'mb-[4px]',
+          ]"
+          :text="$t('AirDrop.mission.decay.minutes')"
+          preset="h2"
+        />
+      </div>
+      <Label
+        :class="[
+          'font-bold',
+          'text-medium-gray',
+        ]"
+        :text="$t('AirDrop.label.decay.start')"
+        preset="p6"
+      />
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator'
+
+@Component
+export default class AirdropDashboard extends Vue {
+
+  // The total claimable amount of airdrop
+  @Prop(Object) readonly totalAirdrop!: object | undefined
+
+  // The total claimed amount of airdrop
+  @Prop(Object) readonly totalClaimedAmount!: object | undefined
+
+  // Contains the time information about decay
+  @Prop(Object) readonly decay!: object | undefined
```

Comment:
I think you are relying on the implicit `toString()` method of bignumber, a safer way to use it is call `toFixed()` or `integerValue()`. You can either call them outside this component and pass them as `number`/`string`, or keep passing them as bignumber object, and call these function in template/computed

Reply:
I see, I thought I had to keep the amount as bignumber object all the time.